### PR TITLE
manifest: Updating nrfxlib - 2022 jan 18

### PR DESCRIPTION
**Pull request by AyturkDuzen**
Updating nrfxlib and mpsl to latest version to fix bugs in periodic advertising and adding periodic advertising list feature

### CODE REVIEW

File: subsys/bluetooth/controller/Kconfig [hunk 1]
```diff
@@ -120,6 +120,17 @@ config BT_CTLR_SDC_PERIODIC_SYNC_BUFFER_COUNT
 	  The buffers are used for processing incoming packets
 	  and storing periodic advertising reports.
 
+# CONFIG_BT_CTLR_SYNC_PERIODIC_ADV_LIST_SIZE is declared in Zephyr and also
+# here for a second time, to set different range for Nordic BLE controller.
```
Comment (rugeGerritsen) [inline subsys/bluetooth/controller/Kconfig:124]:
```suggestion
# here for a second time, to set different range for the SoftDevice Controller.
```
Applies to some other places as well

**Review by rugeGerritsen — CHANGES_REQUESTED** [review]
Mostly looks good. One minor comment. 
You also need to look into the failing compliance check

**Review by rugeGerritsen — APPROVED** [review]

**Review by olivier-le-sage — APPROVED** [review]

File: west.yml [hunk 2]
```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: cadb5102a7ffa9b188960fa3f83a7b2e4351c2bf
+      revision: pull/622/head
```
Comment (rlubos) [inline west.yml:107]: Please update SHA: `ab4cf70a277e44a4b8bc43bcfe9ac8c51f71a03f`

File: subsys/bluetooth/controller/Kconfig (hunk 1 above)
Comment (AyturkDuzen) [reply subsys/bluetooth/controller/Kconfig]: updated

File: west.yml (hunk 2 above)
Comment (AyturkDuzen) [reply west.yml]: updated

**Review by rlubos — APPROVED** [review]

**Pull request merged**